orttraining/orttraining/models/runner/training_runner.cc

```diff
@@ -692,25 +676,26 @@ Status TrainingRunner::RunWithoutUpdate(VectorString& feed_names,
 
   // Async launch of a session.
   pipeline_worker_pool_.workers[worker_id] = std::thread([&](
-      const size_t worker_id, const size_t step) {
+                                                             const size_t worker_id, const size_t step) {
```

Comment:
is this space shift intentional?

Reply:
it is done automatically by my VS clang formator.